keps/sig-network/784-kube-proxy-component-config/README.md

```diff
@@ -127,6 +131,33 @@ The mitigations to those risks:
 
 ## Design Details
 
+### Changes in structure
+
+| v1alpha1               | v1alpha2                | Comments                                                                    |
+|------------------------|-------------------------|-----------------------------------------------------------------------------|
+|                        | Linux                   | new section for linux (platform-specific) options                           |
+| Conntrack              | Linux.Conntrack         | moved from root to linux (platform-specific) section                        |
+| OOMScoreAdj            | Linux.OOMScoreAdj       | moved from root to linux (platform-specific) section                        |
+| IPTables.MasqueradeBit | Linux.MasqueradeBit     | moved from iptables (backend-specific) to linux (platform-specific) section |
+| NFTables.MasqueradeBit | Linux.MasqueradeBit     | moved from nftables (backend-specific) to linux (platform-specific) section |
```

Comment:
I'm not sure about this one; it's really "netfilter-specific", not "linux-specific". An OVS-based or eBPF-based plugin wouldn't use this field (not that we're likely to end up with an OVS-based or eBPF-based proxy implementation in tree at this point, so maybe that's a silly objection). Though also, I have [plans to get rid of the masqueradeBit in the nftables implementation](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/3866-nftables-proxy/README.md#masquerading).

So maybe keep `IPTables.MasqueradeBit` and (for now) `NFTables.MasqueradeBit`, and add `IPVS.MasqueradeBit`?

Reply:
Can we add `Netfilter` section under `Linux`?

`Conntrack` can also go there along with`MasqueradeBit`?